test/Jsr292/src/com/ibm/j9/jsr292/LookupInTests.java

```diff
@@ -356,22 +345,21 @@ public void testLookup_ProtectedInnerClassLookup_Subclass() throws Throwable {
 	 */
 	@Test(groups = { "level.extended" })
 	public void testLookup_ProtectedInnerClassLookup_SamePackage() throws Throwable {
-		Lookup lookup = SamePackageExample2.getLookup();
-		Lookup inObj = lookup.in( SamePackageInnerClass_Protected.class );
-		assertClassAndMode( inObj, SamePackageInnerClass_Protected.class, PUBLIC_PACKAGE_MODE );
+		Lookup inObj = samePackageExample2Lookup.in(SamePackageInnerClass_Protected.class);
+		assertClassAndMode(inObj, SamePackageInnerClass_Protected.class, PUBLIC_PACKAGE_MODE);
 	}
 	
 	/**
 	 * Validates access restrictions stored in a new Lookup object created from an old Lookup object
 	 * where the new lookup class is a protected inner class inside a top level class belonging to a different 
 	 * package than the old lookup class.
+	 * Note: a protected class is treated as public during compilation
```

Comment:
`during compilation`?  Not sure what this means.  

Do you mean `At runtime, the JVM treats a protected class as public`?

Reply:
Updated the misleading comments.